packages/react-notion-x/src/utils.ts

```diff
@@ -82,3 +82,82 @@ export const getYoutubeId = (url: string): string | null => {
 
   return null
 }
+
+export const getWeeksInMonth = (
```

Comment:
This seems to have some complex logic; maybe add a basic comment to describe the function?